### DISCUSSION TITLE
啥時候給更新?

### DISCUSSION THREAD

**Discussion by Tsaitou7361** [Meloong-Git/PCL#1743] [提问] [answered]
我看那源代碼庫都更新到2.5.4了
但最新版本.zip 跟 愛發電給的雲盤地址 都還在 2.5.2 版啊

**Comment by Pigeon0v0** [comment]
正式版还没更新，现在是快照版更新

**Comment by MoYuan-CN** [comment]
PCL2 的源代码库包含 正式版 和 快照版 两种的源码
如果您是**正式版**用户，目前最新的正式版为 Release 2.5.2（287.0） ，请等待后续的更新
如果您是**内测版**用户，目前最新的内测版为 Snapshot 2.5.4（289.0）。若您的启动器没有自动更新，请在 `设置->启动器` 中点击 `检查更新` 按钮来更新启动器

**Comment by Chiloven945** [comment]
正式版目前最新的版本號就是2.5.2，測試版會在沒有問題之後同步至正式版，要獲取測試版鏈接的話需要給龍貓打賞一下www

**Reply by Tsaitou7361** [reply]
我這邊沒辦法給中國那邊打錢QQ

**Comment by WForst-Breeze** [comment, marked as answer]
对于 Release 正式版（即您说的爱发电云盘中和 最新正式版.zip 的版本），一般会在快照版相关功能稳定并基本确认无重大崩溃及功能和使用性漏洞后会同步至正式版。您可以关注 Github 仓库标记为“处理中”的 Issues 或关注哔哩哔哩更新日志专栏来等待正式版更新。

如果您迫切想要抢先体验新功能，可以在爱发电赞助开发者以获取 Snpashot 快照版本（即 Github Releases 中的版本）

**Reply by zkitefly** [reply]
~这很客服~

**Reply by zkitefly** [reply]
qwq

**Comment by OStarsO** [comment]
> 我看那源代碼庫都更新到2.5.4了 但最新版本.zip 跟 愛發電給的雲盤地址 都還在 2.5.2 版啊

所以说什么时候更新

**Reply by Tsaitou7361** [reply]
idk
